draft-privacypass-rate-limit-tokens.md

```diff
@@ -715,13 +698,15 @@ blinded_msg, blind_inv = rsabssa_blind(pkI, token_input)
 The Client then uses Client Key to generate its one-time-use request public
 key `request_key` and blind `request_blind` as described in {{client-anon-issuer-origin-id}}.
 
-The Client then constructs a InnerTokenRequest value, denoted `origin_token_request`,
-combining `blinded_msg`,  `request_key`, and the origin name as follows:
+The Client then computes `token_key_id` as SHA256(public_key), where public_key is a DER-encoded
+SubjectPublicKeyInfo object carrying Token Key. The Client then constructs a InnerTokenRequest
+value, denoted `origin_token_request`, combining `token_key_id`, `blinded_msg`, and a padded
+representation of the origin name as follows:
 
 ~~~
 struct {
+  uint8_t token_key_id[32];
```

Comment:
What about removing the key ID altogether? It will prevent any correlation and key targeting attack concerns.

It does mean that during key rotation, you will have failures. But since we say you shouldn't rely on these tokens, that's probably OK?